go.mod

```diff
@@ -1,13 +1,68 @@
 module github.com/terraform-providers/terraform-provider-pagerduty
 
-go 1.16
+go 1.18
```

Comment:
Switching to go 1.18 is required by newer hcl v2 version:

Without this:
```
make testacc TEST=./pagerduty/ TESTARGS='-run=TestAccPagerDutyService_AlertContentGrouping'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./pagerduty/ -v -run=TestAccPagerDutyService_AlertContentGrouping -timeout 120m
# github.com/hashicorp/hcl/v2
vendor/github.com/hashicorp/hcl/v2/diagnostic_typeparams.go:22:22: type parameter requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/github.com/hashicorp/hcl/v2/diagnostic_typeparams.go:22:24: undeclared name: any (requires version go1.18 or later)
FAIL    github.com/terraform-providers/terraform-provider-pagerduty/pagerduty [build failed]
FAIL
make: *** [GNUmakefile:17: testacc] Error 2
```

See https://github.com/hashicorp/hcl/commit/c3b671524ef9402b2837c2fd14a4ed115fc69772#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R3
and https://github.com/hashicorp/hcl/commit/88ecd1315610ac2c14d5dba8de0a25e75cdd4a58#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R3